types/johnny-five/index.d.ts

```diff
@@ -314,92 +391,93 @@ export interface HygrometerOption {
     freq?: number | undefined;
 }
 
-export declare class Hygrometer {
+export class Hygrometer {
     constructor(option: HygrometerOption);
 
     id: string;
     readonly relativeHumidity: number;
     readonly RH: number;
 
-    on(event: string, cb: () => void): this;
-    on(event: "change", cb: () => void): this;
-    on(event: "data", cb: (data: any) => void): this;
+    on(event: 'change', cb: () => void): this;
+    on(event: 'data', cb: (data: unknown) => void): this;
 }
 
+// tslint:disable-next-line:interface-name
 export interface IMUGeneralOption {
     controller?: string | undefined;
     freq?: number | undefined;
 }
 
+// tslint:disable-next-line:interface-name
 export interface IMUMPU6050Option extends IMUGeneralOption {
     address: number;
 }
 
-export declare class IMU {
+export class IMU {
     constructor(option: IMUGeneralOption | IMUMPU6050Option);
 
     readonly accelerometer: Accelerometer;
-    readonly compass: Compass
+    readonly compass: Compass;
     readonly gyro: Gyro;
-    readonly orientation: Orientiation;
+    readonly orientation: Orientation;
     readonly thermometer: Thermometer;
 
-    on(event: string, cb: () => void): this;
-    on(event: "change", cb: () => void): this;
-    on(event: "data", cb: (data: any) => void): this;
-    on(event: "calibrated", cb: () => void): this;
+    on(event: 'change' | 'calibrated', cb: () => void): this;
+    on(event: 'data', cb: (data: unknown) => void): this;
 }
 
-export declare module IR {
+export interface ArrayOption {
```

Comment:
@scottgonzalez What do you suggest ? Maybe can we so this in another merge request so you can give me more context ?

Reply:
I just meant that `ArrayOption` should be named `ReflectanceArrayOption` since it's specifically the options for `ReflectanceArray` and not some generic set of options for all array components.

Reply:
Done